packages/@coorpacademy-components/src/organism/brand-form/index.js

```diff
@@ -34,26 +32,25 @@ const buildButtonSection = (
       data-for="submitButton"
       className={massiveBattleForm ? style.massiveBattleButton : style.saveButton}
     >
-      {theme === 'coorpmanager' ? (
-        <ButtonLink
-          type="primary"
-          label={submitValue}
-          disabled={disabledSubmit}
-          data-testid="submit-button"
-          buttonType="submit"
-          onClick={onSubmit}
-        />
-      ) : (
-        <Button type="submit" disabled={disabledSubmit} submitValue={submitValue} />
-      )}
+      <ButtonLink
+        type="primary"
+        label={submitValue}
+        disabled={disabledSubmit}
+        data-testid="submit-button"
+        buttonType="submit"
+        onClick={onSubmit}
+      />
     </div>
   ) : null;
 
-  const cancelBackground = {backgroundColor: darkColor};
-
   const resetButton = onReset ? (
     <div className={style.resetButton}>
-      <Button type="reset" submitValue={resetValue} style={cancelBackground} />
+      <ButtonLink
+        buttonType="reset"
```

Comment:
We can't `type` because there is already one, right?
💭  Maybe instead of `buttonType` we could go for something else like `action` or `usage`